### PR TITLE
Updated README and added screenshot to podspec

### LINKED ISSUE #57: readme says 3 additional params in `play()`, there are 4

### LINKED ISSUE #59: fix broken images on CocoaPods page, add screen shot
https://cocoapods.org/pods/SwiftVideoBackground

You can add a screen shot from the Podspec. It will display on the right side bar on the CocoaPods page. I believe this can also be a gif, in which the screenshot could be the Spotify thing.

a reference: https://cocoapods.org/pods/GMStepper

### LINKED ISSUE #60: Add `pod spec lint` to Travis

### PR DESCRIPTION
**Pull request by dingwilson**
## What does this PR do?

Fixes a typo in README
Adds screenshot to podspec

## What issues (if any) are related to this PR? Or why was this change introduced?

Fixes #57 
Fixes #59 
Fixes #60

## Checklist

 - [ ] Does this contain code changes?
 - [ ] Does this have tests?
 - [ ] Does this have documentation?
 - [ ] Does this break the public API (Requires major version bump)?
 - [ ] Is this a new feature (Requires minor version bump)?

### CODE REVIEW

**Review by quanvo87 — DISMISSED** [review]

File: SwiftVideoBackground.podspec [hunk 1]
```diff
@@ -3,6 +3,7 @@ Pod::Spec.new do |s|
   s.version      = "3.0.0"
   s.summary      = "An easy to use Swift framework to play a video in the background of any UIView."
   s.description  = "SwiftVideoBackground is an easy to use Swift framework that provides the ability to play a video on any UIView. This provides a beautiful UI for login screens, or splash pages, as implemented by Spotify and many others"
+  s.screenshots  = "https://github.com/dingwilson/SwiftVideoBackground/raw/master/Assets/Spotify.gif"
```
Comment (quanvo87) [inline SwiftVideoBackground.podspec]: I did a `pod spec lint` and got 
```
- WARN  | screenshot: The screenshot https://github.com/dingwilson/SwiftVideoBackground/raw/master/Assets/Spotify.gif is not a valid image.
```

Comment (quanvo87) [reply SwiftVideoBackground.podspec]: I don't freaking understand why every link I try just downloads the gif and not displays it.

**Review by quanvo87 — CHANGES_REQUESTED** [review]

File: SwiftVideoBackground.podspec (hunk 1 above)
Comment (dingwilson) [reply SwiftVideoBackground.podspec]: Nice catch! Hmm thats weird, I'm using that same link rn for the README and it works just fine.. Maybe I have to change it to read from their raw.githubusercontent link.. I'll try that.

Comment (dingwilson) [reply SwiftVideoBackground.podspec]: Even if I use https://raw.githubusercontent.com/dingwilson/SwiftVideoBackground/master/Assets/Spotify.gif

it still fails to lint (throws the same warning). What I (also) noticed is that the other lib's gif directly plays in the browser, whereas mine seems to download... I can't seem to figure out why.

https://raw.githubusercontent.com/gmertk/GMStepper/master/Screenshots/screenshot_1.gif

Comment (quanvo87) [reply SwiftVideoBackground.podspec]: Yea...maybe just host the gif somewhere else as well?

Comment (dingwilson) [reply SwiftVideoBackground.podspec]: dang I just about to throw the gif onto my website, but then I realized thats run on GitHub too. 😂 Do you have any suggestions/preference for a good hosting site?

Comment (quanvo87) [reply SwiftVideoBackground.podspec]: How about https://imgur.com/

Comment (dingwilson) [reply SwiftVideoBackground.podspec]: Done. It passes pod spec lint now. 🎉

Comment (dingwilson) [reply SwiftVideoBackground.podspec]: I should probably add linting podspec as a check in travis..

**Comment by dingwilson** [comment]
I removed testing for iOS 10 and 9. I think as long as we test the latest and oldest supported version, we should be okay. This will save us significant travis build time.

**Review by quanvo87 — APPROVED** [review]

**Pull request merged**